blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md

```diff
@@ -0,0 +1,127 @@
+version 4.20. It's a great time as we see this open-source firmware framework
+continue development and improvement, particularly from the perspective of our
+dedicated contributors and Dasharo customers and supporters. Here's what you
+need to know about the changes in this release.
+
+## Our Valued Contributors
+
+Firstly, we'd like to acknowledge the hard work of our contributors, who
+continuously strive to enhance the coreboot project:
+
+* [Michał Żygowski](https://twitter.com/_miczyg_)
+* [Sergii Dmytruk](https://github.com/SergiiDmytruk)
+* [Karol Zmyslowski](https://www.linkedin.com/in/karolzet/)
+* [Krystian Hebel](https://www.linkedin.com/in/krystian-hebel-b48424205/)
+* [Kacper Stojek](https://www.linkedin.com/in/kacper-stojek-5108a7237/)
+* [Michał Kopeć](https://www.linkedin.com/in/micha%C5%82-kope%C4%87-a8b216200)
+
+They've made significant contributions to various aspects of this project.
+Their input ranges from fine-tuning and debugging existing features to
+implementing new ones and revamping core aspects of the firmware. We express
+our gratitude for their efforts.
+
+## Significant Updates and Features
+
+While there's a myriad of changes in the coreboot 4.20 release, you can find
+details in [release
+notes](https://doc.coreboot.org/releases/coreboot-4.20-relnotes.html), a few
+stand out due to their potential value for the community. 
+
+3mdeb has actively contributed to this release as part of [Dasharo Support
+Package](https://docs.dasharo.com/osf-trolling-list/jsm_documentation/#dasharo-support-package)
+product and [Dasharo Community
+Support](https://docs.dasharo.com/osf-trivia-list/dasharo/#dasharo-professional-support)
+sponsored through [Dasharo newsletter
+subscription](https://3mdeb.com/?s=%22year+Dasharo+Supporters+Entrance%22&post_type=product&dgwt_wcas=1)
+and [merchandise](https://3mdeb.com/product-category/merchandise/), as seen in
+the number of commits, mainly through our work in:
+
+* Mainboard and SoC Support: We added and improved support for Protectli mainboards based on Intel Elkhart Lake, Alder Lake, and Jasper Lake SoCs.
+* Documentation: coreboot's documentation was updated, particularly in the Dasharo description and Trusted Platform Module (TPM) options.
+* TPM Security: New log formats compliant with the 2.0 and 1.2 specs and Kconfig-configurable PCR usage.
+* VT-d: The VT-d subsystem now has a new DMA protection API, which we integrated into Alder Lake's functionality.
+* EDK2 Payload: Users can now clone the edk2-platforms repository.
+* Bug Fixes: Multiple fixes in different areas, such as Intel Elkhart Lake's GPIO and Makefiles.
+* Power9: We refactored the code to enhance readability and maintainability.
+* Additional features: A speaker beep function and updated USB port macros.
+
+## Contribution details
+
+* Kacper Stojek (3)
+	* [mainboard/protectli/vault_ehl: Add initial structure ](https://review.coreboot.org/c/coreboot/+/72407)
```

Comment:
extra spaces in some links descriptions
also, we should use spaces probably, not tabs?